internal/util/util.go

```diff
@@ -292,3 +296,12 @@ func HasBackupLabel(o *metav1.ObjectMeta, backupName string) bool {
 	}
 	return o.Labels[velerov1api.BackupNameLabel] == label.GetValidName(backupName)
 }
+
+// We expect VolumeSnapshotMoverEnv to be set once when container is started.
```

Comment:
lets create a `datamover.go` under util and add add datamover related stuff there, will be easier to deal with whenever we rebase off upstream